src/test/java/K_pages/AllSongsPage.java

```diff
@@ -0,0 +1,40 @@
+package K_pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+public class AllSongsPage extends BasePage{
+ @FindBy(css=".playlist:nth-child(3)")
+WebElement playlistLocator;
+protected @FindBy(css="input[name='name']")
+   WebElement playlistInputFieldLocator;
+    public AllSongsPage (WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    public void doubleClickOnPlaylist(){
+        doubleClick(playlistLocator);
+    }
+
+    public  void enterPlaylistNewName (String playlistName) {
+        WebElement playlistInputField = driver.findElement((By) playlistInputFieldLocator);
+        playlistInputField.sendKeys((Keys.chord(Keys.chord(Keys.COMMAND, "a",Keys.BACK_SPACE))));
+        playlistInputField.sendKeys(playlistName);
+        playlistInputField.sendKeys(Keys.ENTER);
+        return this;
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 
![image](https://user-images.githubusercontent.com/104761482/229270767-4afcb24b-5935-41dc-87ec-e5346456cdc3.png)

HINT

Delete **return this;**

![image](https://user-images.githubusercontent.com/104761482/229270791-802018d1-019e-4f72-8c33-233b2aa74063.png)